specification/dartLangSpec.tex

```diff
@@ -30,6 +30,8 @@
 % - Allow generic instantiation of expressions with a generic function type
 %   (until now, it was an error unless the expression denoted a declaration).
 % - Allow generic instantiation of the `call` method of a function object.
+% - Disallow `TypeLiteral.extensionMethod()`, in line with the error for
```

Comment:
Maybe say that we make it *explicit* that it was already disallowed.
I think it was. The alternative was that the language was ambiguous and `C.m` could mean *both* a static member and an extension member, and we didn't provide any disambiguation.